eng/Version.Details.xml

```diff
@@ -35,45 +35,37 @@
     <Dependency Name="Microsoft.WindowsAppSDK" Version="0.0.1">
       <Uri>https://dev.azure.com/microsoft/ProjectReunion/_git/ProjectReunionInternal</Uri>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.App.Runtime.win-x64" Version="6.0.3" CoherentParentDependency="Microsoft.Dotnet.Sdk.Internal">
+    <Dependency Name="Microsoft.AspNetCore.Authorization" Version="6.0.4">
```

Comment:
GitHub is choosing to show the diff a bit strangely. So I'll summarize:

Removed:
- Microsoft.AspNetCore.App.Runtime.win-x64
- Microsoft.WindowsDesktop.App.Runtime.win-x64

Also removed the coherant dependencies.

Updated versions to 6.0.4 and updated the hash accordingly.